build/fixtures/README.md

```diff
@@ -43,18 +132,68 @@ define( [ "jquery" ], function( $ ) {
 } );
 ```
 
-### Node
+Node.js doesn't understand AMD natively so this method is mostly use in a browser setup.
 
-To include jQuery in [Node](https://nodejs.org/), first install with npm.
+#### Individual AMD modules
 
-```sh
-npm install jquery
+All ECMAScript modules from `src/` - described in the [Individual modules](#individual-modules) section - have their AMD equivalents in the `amd/` folder.
+
+Similarly to `src/`, this is considered an advanced & low-level interface that may not be stable even between minor or patch releases.
+
+Example usage:
+
+```js
+define( [
+	"jquery/src/css", // adds the `.css()` method
+	"jquery/src/event" // adds the `.on()` method; pulls "selector" as a dependency
+], function( $ ) {
+	$( ".toggle" ).on( "click", function() {
+		$( this ).css( "color", "red" );
+	} );
+} );
 ```
 
+### Node.js pre-requisites
+
 For jQuery to work in Node, a window with a document is required. Since no such window exists natively in Node, one can be mocked by tools such as [jsdom](https://github.com/jsdom/jsdom). This can be useful for testing purposes.
 
+jQuery checks for a `window` global with a `document` property and - if one is not present, as is the default in Node.js - it returns a factory accepting a `window` as a parameter instead.
+
+To `require` jQuery using this factory, use the following:
+
 ```js
 const { JSDOM } = require( "jsdom" );
 const { window } = new JSDOM( "" );
 const $ = require( "jquery" )( window );
 ```
+
+or, if you use ECMAScript modules:
```

Comment:
Your other examples put the ESM code first. In general I would put ESM first as it’s going to be the most common choice for new users going forward.